add doc for sysvar `tidb_session_alias` (#14725)
pingcap/docs: system-variables.md

- Range: `[128, 9223372036854775807]`, in bytes. The memory format with the units "KB|MB|GB|TB" is also supported.
- After you enable the memory limit, TiDB will terminate the SQL statement with the highest memory usage on the current instance. This variable specifies the minimum memory usage of the SQL statement to be terminated. If the memory usage of a TiDB instance that exceeds the limit is caused by too many sessions with low memory usage, you can properly lower the value of this variable to allow more sessions to be canceled.

### `tidb_session_alias` <span class="version-mark">New in v7.4.0</span>

- Scope: SESSION
- Persists to cluster: No
- Default value: ""

> - You can use this variable to customize the value of the `session_alias` column in the logs related to the current session. This setting affects the logs of multiple nodes during the execution of the statement (including TiKV). The maximum length of this variable is limited to 64 characters, and any exceeding length will be truncated automatically. Spaces at the end of the value will also be removed automatically.

Reviewer: ```suggestion
- You can use this variable to customize the value of the `session_alias` column in the logs related to the current session, which helps identify the session in troubleshooting. This setting affects the logs of multiple nodes involved in the statement execution (including TiKV). The maximum length of this variable is limited to 64 characters, and any character exceeding the length will be truncated automatically. Spaces at the end of the value will also be removed automatically.
```

Reviewer: multiple statements might be executed in one session, so  "the execution of the statement" might be inaccurate

> - You can use this variable to customize the value of the `session_alias` column in the logs related to the current session. This setting affects the logs of multiple nodes during the execution of the statement (including TiKV). The maximum length of this variable is limited to 64 characters, and any exceeding length will be truncated automatically. Spaces at the end of the variable value will also be removed automatically.

Reviewer 2: ```suggestion
- You can use this variable to customize the value of the `session_alias` column in the logs related to the current session. This setting affects the logs of multiple nodes during the execution of the statement (including TiKV). The maximum length of this variable is limited to 64 characters, and any exceeding length will be truncated automatically. Spaces at the end of the value will also be removed automatically.
```

> ### `tidb_session_plan_cache_size` <span class="version-mark">New in v7.1.0</span>

Reviewer 3: ```suggestion
### tidb_session_plan_cache_size <span class="version-mark">New in v7.1.0</span>
```

- Range: `[128, 9223372036854775807]`, in bytes. The memory format with the units "KB|MB|GB|TB" is also supported.
- After you enable the memory limit, TiDB will terminate the SQL statement with the highest memory usage on the current instance. This variable specifies the minimum memory usage of the SQL statement to be terminated. If the memory usage of a TiDB instance that exceeds the limit is caused by too many sessions with low memory usage, you can properly lower the value of this variable to allow more sessions to be canceled.

### tidb_session_alias <span class="version-mark">New in v7.4.0</span>

- Scope: SESSION

> - Persists to cluster: No

Reviewer 4: ```suggestion
- Persists to cluster: No
- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): Yes
```
